res/controllers/Hercules-DJ-Console-4-Mx-scripts.js

```diff
@@ -0,0 +1,622 @@
+////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                               //
+////////////////////////////////////////////////////////////////////////
+/* global engine                                                      */
+/* global script                                                      */
+/* global print                                                       */
+/* global midi                                                        */
+//////////////////////////////////////////////////////////////////////// 
+//=====================================================
+//___ Hercules DJ Console 4-Mx scripts by josepma. ____
+//___________ Based partially on the Mk4 script _______
+//=====================================================
+// Version 2015-12-12: Initial version.
+//
+// Usage:
```

Comment:
You don't need to keep this written description here now that there is a wiki page